### PR TITLE
HFA-50 - Spracovanie osobnych udajov podstranka

### PR DESCRIPTION
**Pull request by trofoSK**

### CODE REVIEW

File: src/app/public-view/components/gdpr/components/gdpr-text-section/gdpr-text-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@Component({
+  selector: 'app-gdpr-text-section',
+  templateUrl: './gdpr-text-section.component.html',
+  styleUrls: ['./gdpr-text-section.component.scss']
+})
+export class GdprTextSectionComponent implements OnInit {
+
+  @Input() gdprTextSections: GdprTextSection[];
+
+  constructor() {}
```
Comment (RandomUserIK) [inline src/app/public-view/components/gdpr/components/gdpr-text-section/gdpr-text-section.component.ts]: prazdny constructor a ngOnInit metoda

File: src/app/public-view/components/gdpr/components/models/GdprTextSection.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export interface GdprTextSection{
```
Comment (RandomUserIK) [inline src/app/public-view/components/gdpr/components/models/GdprTextSection.ts]: `GdprTextSection {`

to iste aj nizsie

File: src/app/public-view/components/gdpr/gdpr/gdpr.component.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import data from 'src/app/public-view/components/gdpr/components/models/gdpr-data.json';
+
+@Component({
+  selector: 'app-gdpr',
+  templateUrl: './gdpr.component.html',
+  styleUrls: ['./gdpr.component.scss']
+})
+export class GdprComponent implements OnInit {
+
+  constructor() { }
```
Comment (RandomUserIK) [inline src/app/public-view/components/gdpr/gdpr/gdpr.component.ts]: prazdny constructor a ngOnInit metoda

File: src/app/public-view/components/gdpr/gdpr/gdpr.module.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import {GdprComponent} from './gdpr.component';
+import {GdprTextSectionComponent} from '../components/gdpr-text-section/gdpr-text-section.component';
+
+
+
+@NgModule({
+  declarations: [
+    GdprComponent,
+    GdprTextSectionComponent,
+
```
Comment (RandomUserIK) [inline src/app/public-view/components/gdpr/gdpr/gdpr.module.ts]: newline navyse

File: src/app/public-view/public-view.module.ts [hunk 5]
```diff
@@ -58,6 +60,10 @@ const routes: Routes = [
       {
         path: 'order',
         component: OrderComponent
+      },
+      {
+        path: 'app-gdpr',
```
Comment (RandomUserIK) [inline src/app/public-view/public-view.module.ts]: mozno by som dal rovno len `/gdpr`

**Review by RandomUserIK — CHANGES_REQUESTED** [review]

**Review by RandomUserIK — APPROVED** [review]

File: src/app/public-view/components/footer/footer.component.html [hunk 6]
```diff
@@ -15,7 +15,7 @@
   <div class="row text-center p-2">
     <div class="col">
       <!-- TODO: upravit referenciu na stranku -->
-      <span class="d-block"><a href="#" class="link-text">Zásady spracovania a ochrany osobných údajov</a></span>
+      <span class="d-block"><a href="app-gdpr" class="link-text">Zásady spracovania a ochrany osobných údajov</a></span>
```
Comment (MartinMiklovic) [inline src/app/public-view/components/footer/footer.component.html]: :hankey:

File: src/app/public-view/components/gdpr/components/models/GdprTextSection.ts [hunk 7]
```diff
@@ -0,0 +1,9 @@
+export interface GdprTextSection {
+  title: string;
+  texts: GdprTexts[];
+}
+
+export interface GdprTexts {
```
Comment (MartinMiklovic) [inline src/app/public-view/components/gdpr/components/models/GdprTextSection.ts:6]: :hankey:

**Review by MartinMiklovic — APPROVED** [review]

**Pull request merged**